tensorflow/python/util/deprecation.py

```diff
@@ -107,7 +107,7 @@ def _call_location(outer=False):
   # DISABLE_IMPORT_INSPECT_CHECK=TRUE to your cl description. Using it caused
   # test timeouts (b/189384061).
   f = inspect.currentframe().f_back.f_back
-  parent = f.f_back
+  parent = f and f.f_back
```

Comment:
This would change the type of parent. Try this instead:

```
parent = f.f_back if f is not None else None
```

Reply:
@mdanatg Thank you for your feedback. I am using short-circuiting to do the same thing which you mentioned and I thought it would look cleaner. Please check the screenshot below, I know it is a small change, but I'm not sure how type would change for the parent. Kindly correct me if I'm going wrong.
![Screenshot from 2021-09-14 01-06-35](https://user-images.githubusercontent.com/60918872/133146114-d073a27d-a440-408c-a148-0855f2a6b8f9.png)

Reply:
Ah I forgot about the short circuiting. LGTM then.

Reply:
Yes. Thank you!